db-scheduler/src/main/java/com/github/kagkarlsson/scheduler/task/schedule/CronSchedule.java

```diff
@@ -39,6 +39,8 @@ public class CronSchedule implements Schedule, Serializable {
 
     private static final String DISABLED = "-";
     private static final Logger LOG = LoggerFactory.getLogger(CronSchedule.class);
+
+    private static final long serialVersionUID = 6577185347687546367L;
```

Comment:
Did you need this?

Reply:
Not sure why, but I receive InvalidClassException, if this ID is missing and database entry was created with master version. I used the same JVM for compiling master and this branch, but still the error occurs - not sure why.

```
Caused by: java.io.InvalidClassException: com.github.kagkarlsson.scheduler.task.schedule.CronSchedule; local class incompatible: stream classdesc serialVersionUID = 6577185347687546367, local class serialVersionUID = 6577185347687514335
	at java.base/java.io.ObjectStreamClass.initNonProxy(ObjectStreamClass.java:597) ~[na:na]
	at java.base/java.io.ObjectInputStream.readNonProxyDesc(ObjectInputStream.java:2051) ~[na:na]
	at java.base/java.io.ObjectInputStream.readClassDesc(ObjectInputStream.java:1898) ~[na:na]
	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2224) ~[na:na]
	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1733) ~[na:na]
	at java.base/java.io.ObjectInputStream$FieldValues.<init>(ObjectInputStream.java:2606) ~[na:na]
	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2457) ~[na:na]
	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2257) ~[na:na]
	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1733) ~[na:na]
	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:509) ~[na:na]
	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:467) ~[na:na]
	at com.github.kagkarlsson.scheduler.boot.autoconfigure.DbSchedulerAutoConfiguration$2.deserialize(DbSchedulerAutoConfiguration.java:232) ~[db-scheduler-spring-boot-starter-master-SNAPSHOT.jar:na]
```

Reply:
Hmm, that's a bit strange. When I think about it, I suppose the `serialVersionUID` really should be set in db-scheduler for everything that might be serialized.

Reply:
I think you need to investigate that a bit more. We cannot add that without reason since it might break serialization for other users. That said, it should really have a `serialVersionUID` set..

Reply:
It was a mistake on my part not setting a value for `serialVersionUID` for this class. Serialization will break for anyone using this class when we set it. The solution will be to migrate away from the class temporarily.